projects/project-player-library/src/lib/pages/details-page/details-page.component.css

```diff
@@ -109,6 +114,20 @@
     margin: 10px 0 10px 16px;
     color: rgb(99, 98, 98);
   }
+  ::ng-deep .mat-mdc-tab:not(.mdc-tab--stacked){
```

Comment:
what are these styles added for?

Reply:
this is for the height of the tab in the details page